stubs/sublime.pyi

```diff
@@ -266,6 +268,10 @@ def list_syntaxes() -> Sequence[Dict[str, Any]]:
     ...
 
 
+def command_url(cmd: str, args: Optional[dict] = ...) -> str:
```

Comment:
This wasn't added until 4075 I think?

Does this mean we are forced to make a 4075-x.y.z branch now?

Reply:
No. Let's not be crazy :)

People who are on ST4 dev builds are on the latest version. There is no reason to be on an older version and there is no reason to keep compatibility with older dev builds. :)